docs/tutorials/pasha/scripts/launch_pasha_nasbench201.py

```diff
@@ -0,0 +1,206 @@
+# PASHA: Efficient HPO with Progressive Resource Allocation
+
+from syne_tune.experiments import load_experiment
+from syne_tune.stopping_criterion import StoppingCriterion
+from syne_tune.tuner import Tuner
+from syne_tune.optimizer.baselines import baselines_dict
+from syne_tune.optimizer.schedulers.hyperband import HyperbandScheduler
+from syne_tune.backend.simulator_backend.simulator_callback import SimulatorCallback
+from benchmarking.definitions.definition_nasbench201 import nasbench201_benchmark, nasbench201_default_params
+from benchmarking.blackbox_repository.simulated_tabular_backend import BlackboxRepositoryBackend
+from benchmarking.blackbox_repository import load
+import pandas as pd
+import logging
+
+
+def load_nb201(dataset_name, nb201_random_seed):
```

Comment:
Why is this function needed?

Reply:
The idea is to put the data loading and processing part into a separate function so that the script becomes more readable (especially the part at the bottom). It should also make it slightly easier to reuse the code. However, it is not necessary for it to be within a function.

Reply:
I see now why you asked about that part. The returned dataframe is actually used within the `analyse_experiment` function, but it was not explicitly passed as an argument to the function, which caused the confusion. I've now improved the code to make it clear. Thanks a lot for asking about it!